okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/MeshProxyInterceptor.java

```diff
@@ -39,14 +39,21 @@
     @Override
     public Response intercept(Chain chain) throws IOException {
         HttpUrl originalUri = chain.request().url();
+
+        StringBuilder host = new StringBuilder(originalUri.uri().getHost());
```

Comment:
can you leave a comment here: 
```
// Not using URL#getAuthority since we are removing the userinfo portion. 
// Note that userinfo is not allowed in http2 :authority headers: https://tools.ietf.org/html/rfc7540#section-8.1.2.3
```